src/site/stages/build/drupal/api.js

```diff
@@ -220,8 +220,12 @@ function getDrupalClient(buildOptions, clientOptionsArg) {
         const json = await request;
 
         if (json.errors) {
-          console.log(json.errors);
-          throw new Error(`${queryName} resulted in errors`);
+          const formattedErrors = JSON.stringify(json.errors, null, 2);
+          const pluralizedErrors =
+            json.errors.length > 1 ? 'errors' : 'an error';
+          throw new Error(
+            `Error with ${queryName}:\n\n${formattedErrors}\n\nGraphQL query that has ${pluralizedErrors}:\n${query}`,
+          );
```

Comment:
I actually wonder if we could make use of `chalk` here at all. Like maybe the error could be printed in red; the GraphQL query in blue; and then the actual error we throw is something basic like "GraphQL error encountered" or something like that

Reply:
![image](https://user-images.githubusercontent.com/12773166/109178991-ac799f80-7746-11eb-9a0c-5eb9110d3357.png)
![image](https://user-images.githubusercontent.com/12773166/109179070-c2876000-7746-11eb-8c4f-2964945ee1b9.png)

Great suggestion, updated! 🙂

Reply:
Whoaaaa 👏